java/src/jmri/jmrit/roster/JmritRosterBundle_fr.properties

```diff
@@ -149,5 +145,20 @@ ThrottleLabels = \u00c9tiquettes de R\u00e9gulateur
 Throttle = R\u00e9gulateur
 Export = Exporter
 Exportddd = Export ...
+Import = Importer
+Importddd = Importer...
+ConfirmImport = Confirmer Importer
+ConfirmImportID = Importer locomotive ID: "{0}"
+ConfirmImportDup = Ceci remplacera une entr\u00e9e existante de l'inventaire\npar le m\u00eame nom:\n"{0}"
+CancelImports = Annuler les Importations
+Skip = Sauter
+OK = OK
 
 NotImplemented = Non impl\u00e9ment\u00e9
+
+# {0} - RosterEntry Id
+# {1} - RosterEntry NomLigne
+# {2} - RosterEntry NuméroLigne
```

Comment:
Sorry for this mistake ,
line 164 #{2} - RosterEntry Num\u00e9roLigne

Le 16/01/2016 18:16, Matthew Harris a écrit :
>
> In java/src/jmri/jmrit/roster/JmritRosterBundle_fr.properties 
> <https://github.com/JMRI/JMRI/pull/625#discussion_r49934450>:
>
> >
> >  NotImplemented = Non impl\u00e9ment\u00e9
> > +
> > +# {0} - RosterEntry Id
> > +# {1} - RosterEntry NomLigne
> > +# {2} - RosterEntry NuméroLigne
>
> This needs native2ascii conversion
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/JMRI/JMRI/pull/625/files#r49934450>.
>

-- 
Hervé BLOREC

"L'expérience et le savoir n'ont de valeurs que s'ils sont partagés"